File: FCUBEAPI/FleetTrans/Models/SparesPurchaseRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class SparesPurchaseRptListModel
    {
        public List<SparesPurchaseRptModel> SparesPurchaseRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/sparespurchaserptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Sparespurchaserptmodel } from "./sparespurchaserptmodel";

export class Sparespurchaserptlistmodel {
    sparesPurchaseRptList: Sparespurchaserptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}